zuul.d/project-templates.yaml

```diff
@@ -758,7 +752,6 @@
         - ansible-test-sanity-docker-stable-2.9
         - ansible-test-sanity-docker-stable-2.10
         - ansible-test-sanity-docker-stable-2.11
-        - ansible-test-units-frr
```

Comment:
looks like we didn't delete this job... also same comment for nxos, we are missing some unit test coverage here.

Reply:
jobs added in https://github.com/ansible/ansible-zuul-jobs/pull/910